### PR TITLE
conky needs lua

### PR DESCRIPTION
**Pull request by curiosityseeker**
See issue #3250

### CODE REVIEW

File: etc/conky.profile [hunk 1]
```diff
@@ -8,6 +8,8 @@ include globals.local
 
 noblacklist ${PICTURES}
 
+noblacklist /usr/lib/liblua*
```
Comment (reinerh) [inline etc/conky.profile]: you could include allow-lua.inc.

Comment (Vincent43) [reply etc/conky.profile]: Yes, even if not all lines from `allow-lua.inc` are needed using includes is cleaner for maintenance than  custom `noblacklist` rules in profile.

Comment (curiosityseeker) [reply etc/conky.profile]: Well, I always try to keep the attack surface as small as possible. But if you prefer to add the inc file I'll change that.

**Comment by rusty-snake** [comment]
Merged, Thanks.

**Pull request merged**